src/plugins/mf_priority.cpp

```diff
@@ -1135,11 +1135,14 @@ static int inactive_cb (flux_plugin_t *p,
     if ((b->held_jobs.size () > 0) && (b->cur_run_jobs < b->max_run_jobs)) {
         long int jobid = b->held_jobs.front ();
 
-        if (flux_jobtap_dependency_remove (p,
-                                           jobid,
-                                           "max-running-jobs-user-limit") < 0)
+        if (flux_jobtap_dependency_remove (
+                                          p,
```

Comment:
Just a formatting nit, I think 

```
        if (flux_jobtap_dependency_remove (p,
```

would be the format more in line with the rest of the project

Reply:
Thanks for catching this! Force pushed up a fix for formatting. Will set MWP